This patch file is from Qualcomm Innovation Center, Inc. and is provided under the following license:

Copyright (c) 2024 Qualcomm Innovation Center, Inc. All rights reserved.
SPDX-License-Identifier: BSD-3-Clause-Clear

From 77b4d4fab04ba2a660e3d2a08e067b0d806ae3e9 Mon Sep 17 00:00:00 2001
From: Dongyuan Li <quic_dongyuan@quicinc.com>
Date: Thu, 28 Nov 2024 16:05:29 +0800
Subject: [PATCH] dhcpcd: Disable auto IP configuration

Disable auto IP configuration on eth0, wlan2 and wlan3

Upstream-Status: Pending

Change-Id: I5d399586055343a42ef72ffb26888d2f81bcccce
Signed-off-by: Dongyuan Li <quic_dongyuan@quicinc.com>
---
 src/dhcpcd.conf | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/src/dhcpcd.conf b/src/dhcpcd.conf
index 916e82da..80f639dc 100644
--- a/src/dhcpcd.conf
+++ b/src/dhcpcd.conf
@@ -46,3 +46,12 @@ require dhcp_server_identifier
 #slaac hwaddr
 # OR generate Stable Private IPv6 Addresses based from the DUID
 slaac private
+
+# Disable auto IP configuration
+denyinterfaces eth0
+denyinterfaces wlan2
+denyinterfaces wlan3
+denyinterfaces vlan40
+denyinterfaces vlan41
+denyinterfaces vlan42
+denyinterfaces vlan43
-- 
2.34.1

